prac_05/word_occurrences.py

```diff
@@ -0,0 +1,20 @@
+"""
+Write a program to count the occurrences of words in a string. The program should ask the user for a string,
+then print the counts of how many of each word are in the file.
+"""
+
+string_to_count = (input("Please enter a string to count: ")).lower().split(" ")
```

Comment:
string_to_count could just be text like the prompt in the output? "Text: this is a collection of words..."